File: HrmsAngular/src/app/models/leave/earn-leave-balance.model.ts
import { EarnLeaveBalanceDetails } from "./earn-leave-balance-details.model";

export class EarnLeaveBalance{
    id:number;
    lType:number;
    yearID:number;
    date:any;
    note:string;
    companyID:number;
    details:EarnLeaveBalanceDetails[];
    empCode:string;
    qty:number;
    department:string;
    designation:string;
    jobLocation:string;
    grade:number;
    executeType:number;

}

File: HrmsWebApiCore/WebApiCore/Models/Leave/EarnLeaveBalanceModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using WebApiCore.Models.Leave;

namespace HRMS.Models.Leave
{
    public class EarnLeaveBalanceModel
    {
        public int? ID { get; set; }
        public int LType { get; set; }
        public int YearID { get; set; }
        public DateTime DATE { get; set; }
        public string Note { get; set; }
        public int CompanyID { get; set; }
        public List<EarnLeaveBalanceDetailsModel> Details { get; set; }
    }
}